frog/imports/api/objects.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+// $FlowFixMe
+import { Mongo } from 'meteor/mongo';
+import type { ObjectT } from 'frog-utils'; // eslint-disable-line
```

Comment:
They changed it so you can do 

import {john, type Peter} from 'xxx'

however there is a problem with prettier (See my email) for now. Will be fixed soon I'm sure.